mobile/ios/Runner.xcodeproj/xcshareddata/xcschemes/airqodev.xcscheme

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@Baalmart , this is the file contains all the necessary information, concerning the airqodev scheme.